content/community/how-to/join-community-chat.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Join the community chat
+aliases:
+  - /community/how-to-join-mattermost/
+  - /community/how-to/join-mattermost/
+linkTitle: Join the chat
+---
+
+# How to join the community chat
+
+The Open Terms Archive community concentrates interactions on an instant chat platform where every contributor and user is welcome. The following how-to guide will help you get into the community chat.
+
+## 1. Create an account
+
+**Use this [invitation link](https://community.opentermsarchive.org/signup_user_complete/?id=fb1tb45hnfnk3c88btjhsjoomc&md=link&sbr=fa) and create an account.**
+
+After successful login, you will be automatically be added to the **~announcements** channel.
```

Comment:
Why using `~` before channel name? Is it common convention?

Reply:
Yes.